### PR TITLE
Fix validation of null values.

### PR DESCRIPTION
**Pull request by SebastianStehle**

### CODE REVIEW

File: src/NJsonSchema/Validation/JsonSchemaValidator.cs [hunk 1]
```diff
@@ -50,9 +50,10 @@ public JsonSchemaValidator(params IFormatValidator[] customValidators)
         /// <summary>Validates the given JSON data.</summary>
         /// <param name="jsonData">The json data.</param>
         /// <param name="schema">The schema.</param>
+        /// <param name="schemaType">The type of the schema.</param>
         /// <exception cref="JsonReaderException">Could not deserialize the JSON data.</exception>
         /// <returns>The list of validation errors.</returns>
-        public ICollection<ValidationError> Validate(string jsonData, JsonSchema schema)
+        public ICollection<ValidationError> Validate(string jsonData, JsonSchema schema, SchemaType schemaType = SchemaType.OpenApi3)
```
Comment (RicoSuter) [inline src/NJsonSchema/Validation/JsonSchemaValidator.cs:56]: I'd say the default should be JsonSchema and not OpenApi3 as this is a JSON Schema library first.

Comment (SebastianStehle) [reply src/NJsonSchema/Validation/JsonSchemaValidator.cs]: Done

**Comment by RicoSuter** [comment]
Can we merge?

**Comment by SebastianStehle** [comment]
From my side: yes

**Comment by RicoSuter** [comment]
Ok, thanks for the PR.

**Pull request merged**